theme/templates/resource-landing-page/modals.html

```diff
@@ -0,0 +1,773 @@
+                  method="POST" enctype="multipart/form-data">
+                {% csrf_token %}
+                <input name="resource-mode" type="hidden" value="edit">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                    <h4 class="modal-title" id="myModalLabel"> Edit Relation </h4></div>
+                <div class="modal-body">
+                    {% csrf_token %}
+
+                    <div class="form-group">
+                        {% csrf_token %}
+                        <fieldset>
+                            <legend>Relation</legend>
+                            <div id="div_id_type" class="control-group">
+                                <label for="id_type" class="control-label requiredField">
+                                Relation type<span class="asteriskField">*</span></label>
+
+                                <div class="controls">
+                                    <select class="form-control input-sm select" id="id_relation-{{ forloop.counter0 }}-type" name="relation-{{ forloop.counter0 }}-type">
+                                        <option value="">---------</option>
+                                        {% for type_value, type_display in relation_source_types %}
+                                            <option value={{ type_value }} {% if relation.type == type_value %}selected="selected"{% endif %}>{{ type_display }}</option>
+                                        {% endfor %}
+                                    </select>
+                                </div>
+                            </div>
+                            <div id="div_id_value" class="control-group">
+                                <label for="id_value" class="control-label requiredField">
+                                Related to<span class="asteriskField">*</span></label>
+
+                                <div class="controls"><input
+                                        class="form-control input-sm textinput textInput" id="id_relation-{{ forloop.counter0 }}-value"
+                                        maxlength="500" name="relation-{{ forloop.counter0 }}-value" type="text"
+                                        value="{{ relation.value }}"></div>
+                            </div>
+                        </fieldset>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                    <button type="submit" class="btn btn-primary">Save changes</button>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+{% endfor %}
+
+{% for contributor in contributor_formset.initial %}
+<div class="modal fade" id="edit-contributor-dialog{{ contributor.id }}" tabindex="-1" role="dialog"
```

Comment:
Similarly instead of 2 different edit modal dialogs for creator and contributor, it should be possible to have one modal dialog ```edit_person_modal.html``` that then needs to be included twice in this file.